nbs/docs/Prediction_intervals_in_forecasting_models.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<p>We've run into situations where we get a diff in the notebooks because of the plot sizes, so what we do now is save them in the nbs/figs folder with the name of the notebook as the prefix, double underscore ("__") and then a representative name for the plot. So for example this could be:</p><pre class="ql-syntax" spellcheck="false">fig = plot_series(df)
fig.savefig('../figs/prediction_intervals_in_forecasting_models__eda.png')
</pre><p>and then you show it in the next cell by making it a markdown cell and referencing it, like:</p><pre class="ql-syntax" spellcheck="false">![](../figs/prediction_intervals_in_forecasting_models__eda.png)
</pre><p>Can you please make those changes for all plots?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/mlforecast/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>